app/src/main/res/layout/item_tab_details.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <androidx.constraintlayout.widget.ConstraintLayout xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/item_view"
+        android:layout_width="160dp"
+        android:layout_height="148dp"
+        android:layout_margin="10dp">
+
+        <ImageView
+            android:id="@+id/item_image"
+            android:layout_width="match_parent"
```

Comment:
you didn't handle gradient view like zeplin

Reply:
👍